beacon_node/lighthouse_network/src/service/mod.rs

```diff
@@ -1449,6 +1455,8 @@ impl<AppReqId: ReqId, TSpec: EthSpec> Network<AppReqId, TSpec> {
                     debug!(self.log, "Failed to dial address"; "peer_id" => ?peer_id,  "error" => %error);
                     None
                 }
+                // We still need to handle this during the deprecation period anyway.
```

Comment:
Unless you are using the `Swarm::ban` APIs, you don't need to :)

You can just _ the binding to make the deprecation warning go away.